Makefile

```diff
@@ -206,6 +206,7 @@ util/build_version.cc: FORCE
 	else mv -f $@-t $@; fi
 
 LIBOBJECTS = $(LIB_SOURCES:.cc=.o)
+LIBOBJECTS += $(TOOL_SOURCES:.cc=.o)
```

Comment:
Do we really need to include tool sources in our library? This will increase our library size. Can you just add TOOL_SOURCES under amalgamation?